src/test/reactutils.tsx

```diff
@@ -25,19 +26,28 @@ export function expectError(message: string, fn: () => void) {
   }
 }
 
-// Utility to handle nulls
-export function renderToDom<C extends ComponentType<{}>>(component: ReactElement<C>) {
-    const c = ReactTestUtils.renderIntoDocument(component) as C;
-    if (!c) {
-        throw new Error(`BUG: Component was not rendered`);
-    }
-    const node = ReactDOM.findDOMNode(c) as Element;
-    if (!node || !node.parentNode) {
-        throw new Error(`BUG: Could not find DOM node`);
+// Utility to handle nulls and SFCs
+export function renderToDom<C extends ComponentType<{}>>(subject: ReactElement<C>) {
+
+  // tslint:disable-next-line:variable-name
+  const Wrapper = class extends Component {
```

Comment:
Ah, so a SFC can't be rendered directly... it _always_ needs a `<Component>` around it. Thanks!

Reply:
yea that was a bummer, but i think this will work to just always wrap it